lib/install/validate/validator.go

```diff
@@ -357,6 +359,86 @@ func (v *Validator) generateBridgeName(ctx, input *data.Data, conf *metadata.Vir
 	return input.DisplayName
 }
 
+func (v *Validator) firewall(ctx context.Context) {
+	defer trace.End(trace.Begin(""))
+
+	var hosts []*object.HostSystem
+	var err error
+
+	rule := types.HostFirewallRule{
+		Port:      8080, // serialOverLANPort
+		PortType:  types.HostFirewallRulePortTypeDst,
+		Protocol:  string(types.HostFirewallRuleProtocolTcp),
+		Direction: types.HostFirewallRuleDirectionOutbound,
+	}
+
+	if hosts, err = v.Session.Datastore.AttachedClusterHosts(ctx, v.Session.Cluster); err != nil {
+		log.Errorf("Unable to get the list of hosts attached to given storage: %s", err)
+		v.NoteIssue(err)
+		return
+	}
+
+	var misconfiguredEnabled []string
+	var misconfiguredDisabled []string
+	var correct []string
+
+	for _, host := range hosts {
+		fs, err := host.ConfigManager().FirewallSystem(ctx)
+		if err != nil {
+			v.NoteIssue(err)
+			return
+		}
+
+		disabled := false
+		esxfw, err := esxcli.GetFirewallInfo(host)
+		if err == nil && !esxfw.Enabled {
+			disabled = true
+			log.Infof("Firewall status: DISABLED on %s", host.InventoryPath)
+		} else {
+			log.Infof("Firewall status: ENABLED on %s", host.InventoryPath)
```

Comment:
Right here it's just reporting the status - if the firewall is enabled it's still OK as long as the correct outbound port is open, which is open by default. We check if the port is open further down and then warn if it's not open.